src/Microsoft.AspNet.Mvc.Core/ActionResults/ObjectResult.cs

```diff
@@ -38,6 +38,14 @@ public ObjectResult(object value)
         /// </summary>
         public int? StatusCode { get; set; }
 
+        /// <summary>
+        /// Gets or sets a flag to enable content-negotiation to override formatter match based on
+        /// a request's Accept and Content-Type headers. If set to <c>true</c>, content-negotiation will
```

Comment:
Needs to clarify:

**if we can't find a match for the requests Accept and Content-Type headers** then...